session_0/word_processing.rb

```diff
@@ -18,7 +18,8 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+  return  words.filter_map{|word| word if word.index(prefix)==0}
```

Comment:
`filter_map` function is something I learned today, thanks!

Do you think `starts_with?` would be more clear `index(...) == 0`?

Reply:
Yeah I wasn't able to think of that